### ISSUE TITLE
way to stop shinyStan from shinyStan?

### ISSUE THREAD

**Issue by bob-carpenter** [stan-dev/shinystan#60]
Is there a way to stop shinyStan from shinyStan?

The instructions for Shiny (http://shiny.rstudio.com/articles/running.html) have this:

<blockquote>
To stop the application you simply interrupt R – you can do this by pressing the Ctrl-C in some R front ends, or the Escape key in RStudio,or by clicking the stop button if your R environment provides one.
</blockquote>

Ctrl-C doesn't work in the R GUI (at least on Mac OS X).  I had to just kill the whole session and workspace from the outside.

**Comment by jgabry** [comment]
Yeah there should be a Quit button in the navbar. Is that not working? Or
should I make it more prominent?

**Comment by bob-carpenter** [comment]
I see it now.  I don't think I was expecting a button
when I saw it before.  I'm used to non-browser apps where
the quit's always in the same place.  So I just tried
to close the browser, but that left me not able to interrupt.

Yes, I'd make it bolder and more prominent and perhaps on
the left rather than the right.  But it's no big deal now that
I know where it's at.

I like that you do not pop up a modal dialog confirming
quit.

- Bob
